pywr/recorders/_recorders.pyx

```diff
@@ -1660,16 +1660,21 @@ cdef class AnnualTotalFlowRecorder(Recorder):
     For each scenario, record the total flow in each year across a list of nodes.
     Output from data property has shape: (years, scenario combinations)
 
+    A list of factors can be provided to scale the total flow (e.g. for calculating operational costs).
+
     Parameters
     ----------
     model : `pywr.core.Model`
     name : str
         The name of the recorder
     nodes : list
         List of `pywr.core.Node` instances to record
+    factors (OPTIONAL) : dictionary
+        Dictionary of (node, factor) for each factor to apply to each node
     """
     def __init__(self, model, str name, list nodes, *args, **kwargs):
         temporal_agg_func = kwargs.pop('temporal_agg_func', 'sum')
+        self.factors = kwargs.pop('factors', {})
```

Comment:
It will require you to handle no factors being supplied. In that case I guess it should default a list of 1.0s.